tests/functional/r/regression_02/regression_5244.py

```diff
@@ -8,5 +8,5 @@ class MyClass:
     def some_func(self):
         return lambda: 42
 
-    def __len__(self):
+    def __len__(self) -> int:
```

Comment:
Code in functional tests or test data should not be modified to add typing (there's no value and it might actually be harmful).